_includes/body.html

```diff
@@ -101,6 +104,7 @@ <h2>{{ site.data.trans[page.lang].extensionguide|safe }}</h2>
                 <li><span class="dot-wrapper"><span class="dot easy"></span></span> - {{ site.data.trans[page.lang].guideeasy|safe }}</li>
                 <li><span class="dot-wrapper"><span class="dot medium"></span></span> -  {{ site.data.trans[page.lang].guidemedium|safe }}</li>
                 <li><span class="dot-wrapper"><span class="dot hard"></span></span> - {{ site.data.trans[page.lang].guidehard|safe }}</li>
+                <li><span class="dot-wrapper"><span class="dot only-required"></span></span> - {{ site.data.trans[page.lang].guideonlyrequired|safe }}</li>
                 <li><span class="dot-wrapper"><span class="dot impossible"></span></span> - {{ site.data.trans[page.lang].guideimpossible|safe }}</li>
             </ul>
         </div>
```

Comment:
Needs to account for the new "difficulty" on the difficulties dropdown below (line 146).